src/passes/LogExecution.cpp

```diff
@@ -39,23 +40,47 @@ namespace wasm {
 Name LOGGER("log_execution");
 
 struct LogExecution : public WalkerPass<PostWalker<LogExecution>> {
-  void visitLoop(Loop* curr) { curr->body = makeLogCall(curr->body); }
+  std::map<Function*, Index> functionOrdinals;
```

Comment:
It doesn't look like this needs to be iterated over.
```suggestion
  std::unordered_map<Function*, Index> functionOrdinals;
```